api/models/task.py

```diff
@@ -362,6 +363,21 @@ def verify_config(config_obj, config):
         pass
 
 
+class Audio(AnnotationComponent):
+    @staticmethod
+    def verify(
+        name,
+        config,
+        data,
+        mode=AnnotationVerifierMode.default,
+    ):
+        pass
+
+    @staticmethod
+    def verify_config(config_obj, config):
+        pass
```

Comment:
Don't we need to have verification here? Afaik the parent class method has a pass here because it needs to be implemented by the child?

Reply:
exactly, it is necessary to implement this method for verification that the audio file uploaded when creating an example meets the conditions. The idea is that as this feature will not be available for the first part of the challenge it is in our backlog to implement this method in the near future.